contracts/strategies/rfp-simple/RFPSimpleStrategy.sol

```diff
@@ -16,7 +16,7 @@ contract RFPSimpleStrategy is BaseStrategy, ReentrancyGuard {
     /// ========== Storage =============
     /// ================================
 
-    /// @notice Struct to hold details of an recipient
+    /// @notice Struct to hold details of a recipient
```

Comment:
Don't worry about the natspec here, we have #224 to take care of this.

Reply:
So I should change it back to the original, or just keep it as it is.
Moreover, it kind of looks grammatically good. and if you want me to change it, won't be a big deal !!